src/templates.rs

```diff
@@ -52,6 +60,47 @@ impl Templates {
         }
     }
 
+    #[allow(clippy::too_many_arguments)]
+    pub fn get_body_by_implicit_name(
+        &self,
+        organization_id: Option<&str>,
+        environment_name: Option<&str>,
+        starts_with: Option<&str>,
+        ends_with: Option<&str>,
+        contains: Option<&str>,
+        export: bool,
+        secrets: bool,
+        template_name: &str,
+    ) -> GraphQLResult<Option<String>> {
+        let query = GetImplicitTemplateQuery::build_query(get_implicit_template_query::Variables {
+            organization_id: organization_id.map(|id| id.to_string()),
+            environment_name: environment_name.map(|name| name.to_string()),
+            template_name: template_name.to_string(),
+            filters: Some(get_implicit_template_query::ImplicitTemplateFilters {
```

Comment:
If you're always going to supply a list of filters, you can adjust the query to `$filters: ImplicitTemplateFilters!`, which will eliminate the need to wrap the filters in an `Option`.